string command
int32 distance
int32 angle
float32 poseX
float32 poseY
float32 poseThetaDegrees

